bigframes/ml/llm.py

```diff
@@ -70,9 +86,19 @@ def __init__(
         model_name: Literal["text-bison", "text-bison-32k"] = "text-bison",
         session: Optional[bigframes.Session] = None,
         connection_name: Optional[str] = None,
+        max_iterations: int = 300,
+        evaluation_task: Literal[
```

Comment:
Let's not to expose evaluation_task. We don't auto split and don't want to run auto evaluation. 

User needs to call score() to run evaluation.